vue-frontend/src/components/JobsScreen.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+    <div>
+        <ScreenHeader />
+    </div>
+</template>
+
+<script>
+import ScreenHeader from "./partials/ScreenHeader.vue";
+
+export default {
+  components: {
+      ScreenHeader,
+  },
+  mounted() {
+    window.gtag("event", "view_canopas_jobs");
```

Comment:
Let's not add events now, to make sure we don't pollute real user data. We will add events when removing feature flag.